load.js

```diff
@@ -14,6 +14,19 @@ const ABI = process.versions.modules
 const inWebpack = typeof __webpack_require__ === 'function'
 const runtimeRequire = inWebpack ? __non_webpack_require__ : require
 
+function maybeLoadWASM(name) {
```

Comment:
This could just be `loadWASM` as it would never fail to load since there cannot be a missing platform. It could probably also just be added to `load` so the caller doesn't have to know whether it's WASM.